shell.nix

```diff
@@ -0,0 +1,34 @@
+# You can get a shell with nix-shell --arg ghcjs <true|false>
```

Comment:
I prefer not to keep files which tend to change a lot as time passes, such as `stack.yaml` and `shell.nix` in repos of libraries (as opposed to repos of actual projects, where it is fine). So I'd rather drop this one.

Reply:
Can delete this, no problems.